azure/__init__.py

```diff
@@ -1,7 +1,4 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-from pkgutil import extend_path
-import typing
-
-__path__: typing.Iterable[str] = extend_path(__path__, __name__)
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
Also I realize that this is less clean than using a proper `import` statement.
I'm only going for this since this is exactly what our docs person suggested we replace our previous file with.

Reply:
@davidmrdavid - You need to make flake happy! :)

```python
Traceback (most recent call last):
  File "/home/runner/work/azure-functions-python-library/azure-functions-python-library/tests/test_code_quality.py", line 54, in test_flake8
    f'flake8 validation failed:\n{output}') from None
AssertionError: flake8 validation failed:
./azure/__init__.py:4:65: W292 no newline at end of file
```

Reply:
haha, I thought I'd get away without checking the linter with a 1-line file...oops 😆